test/elixir/test/partition_all_docs_test.exs

```diff
@@ -116,6 +116,43 @@ defmodule PartitionAllDocsTest do
     assert ids == ["foo:22"]
   end
 
+  test "partition all docs can set query limits", context do
+    resp = Couch.get("/_membership")
+    %{:body => body} = resp
+
+    Enum.each(body["all_nodes"], fn node ->
+      resp = Couch.put("/_node/#{node}/_config/query_server_config/partition_query_limit", body: "\"2000\"")
```

Comment:
Re-reading again, and just noticed this. You should use set_config which takes care of removing the config change after the test finishes. There's an example here:

https://github.com/apache/couchdb/blob/8d2af71f2896d5f75ceb0f780993493fcecdf49d/test/elixir/test/partition_size_limit_test.exs#L15

Reply:
And samesies for partition_view_test